maps_dashboards/public/model/ILayerConfig.ts

```diff
@@ -12,4 +12,6 @@ export interface ILayerConfig {
   opacity: number;
   visibility: string;
   layerSpec?: any;
+  source?: any;
+  style?: any;
```

Comment:
Can we type these better?

Reply:
Yes, I defined type `MapLayerSpecification`, `OSMLayerSpecification`, `DocumentLayerSpecification`.